presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -399,6 +399,7 @@ public WindowFunctionSupplier load(SpecializedFunctionKey key)
                 .scalar(MapCardinalityFunction.class)
                 .scalar(MapConcatFunction.class)
                 .scalar(MapToMapCast.class)
+                .scalar(StringFunctions.FirstFiveCharsReversed.class)
```

Comment:
We should not. But there is not other option for now AFAIK. Function list is hardcoded in FR constructor.
Later we can replace this `FirstFiveCharsReversed` with some real function (e.g. one of decimal operators)